python/llama-index-server/llama_index/server/api/routers/chat.py

```diff
@@ -48,10 +54,20 @@ async def chat(
                 workflow = workflow_factory(chat_request=request)
             else:
                 workflow = workflow_factory()
-            workflow_handler = workflow.run(
-                user_msg=user_message.content,
-                chat_history=chat_history,
-            )
+
+            # Check if we should resume a chat with a human response
+            human_response = last_message.human_response
+            if human_response:
+                workflow_handler = await HITLWorkflowService.resume(
+                    id=request.id,
+                    workflow=workflow,
+                    data=human_response,
+                )
```

Comment:
maybe just use updated load_context here (see comment below)
```suggestion
                ctx = await HITLWorkflowService.load_context(
                    id=request.id,
                    workflow=workflow,
                    data=human_response,
                )
                workflow_handler = workflow.run(ctx=context)
```